src/harness/reference_models/iap/test_data/gwpz_2.json

```diff
@@ -0,0 +1,54 @@
+{  
+   "id":"incumbent/uls/2",
+   "type":"PART_90",
+   "deploymentParam":{  
+      "operationParam":{  
+         "operationFrequencyRange":{  
+            "lowFrequency":3690000000.0,
+            "highFrequency":3700000000.0
```

Comment:
Shouldn't the GWPZ bandwidth be either 25MHz or 50 MHz?

Reply:
Removed   gwpz_2.json file as no longer required